docker/cygnus-ngsi/cygnus-entrypoint.sh

```diff
@@ -1005,16 +1018,16 @@ if [ "$CYGNUS_ARCGIS_URL" != "" ]; then
     if [ "$CYGNUS_ARCGIS_BATCH_TTL" != "" ]; then
         sed -i '/#'${CYGNUS_AGENT_NAME}'.sinks.arcgis-sink.batch_ttl/c '${CYGNUS_AGENT_NAME}'.sinks.arcgis-sink.batch_ttl = '${CYGNUS_ARCGIS_BATCH_TTL} ${FLUME_HOME}/conf/${AGENT_CONF_FILE}
     fi
-	if [ "$CYGNUS_ARCGIS_BATCH_TTL" != "" ]; then
+        if [ "$CYGNUS_ARCGIS_BATCH_TTL" != "" ]; then
         sed -i '/#'${CYGNUS_AGENT_NAME}'.sinks.arcgis-sink.batch_ttl/c '${CYGNUS_AGENT_NAME}'.sinks.arcgis-sink.batch_ttl = '${CYGNUS_ARCGIS_BATCH_TTL} ${FLUME_HOME}/conf/${AGENT_CONF_FILE}
     fi
     if [ "${CYGNUS_MULTIAGENT,,}" == "true" ]; then
         if [ "$CYGNUS_MONITORING_TYPE" != "" ]; then
             # Run the Cygnus command with monitoring
-            ${FLUME_HOME}/bin/cygnus-flume-ng agent --conf ${CYGNUS_CONF_PATH} -f ${FLUME_HOME}/conf/${AGENT_CONF_FILE} -n ${CYGNUS_AGENT_NAME} -p 5080 -Dflume.root.logger=${CYGNUS_LOG_LEVEL},${CYGNUS_LOG_APPENDER} -Duser.timezone=UTC -Dfile.encoding=UTF-8 -Dflume.monitoring.type=${CYGNUS_MONITORING_TYPE} -Dflume.monitoring.port=41415 &
+            ${FLUME_HOME}/bin/cygnus-flume-ng agent --conf ${CYGNUS_CONF_PATH} -f ${FLUME_HOME}/conf/${AGENT_CONF_FILE} -n ${CYGNUS_AGENT_NAME} -p 5089 -Dflume.root.logger=${CYGNUS_LOG_LEVEL},${CYGNUS_LOG_APPENDER} -Duser.timezone=UTC -Dfile.encoding=UTF-8 -Dflume.monitoring.type=${CYGNUS_MONITORING_TYPE} -Dflume.monitoring.port=41424 &
```

Comment:
Maybe we should declare ports in variables at the beginning of the .sh file (eg. `CYGNUS_PORT_ARCGIS=5089`) for convenience.

NTC (not for this PR but as idea for a next one)

Reply:
(Maybe related with https://github.com/telefonicaid/fiware-cygnus/issues/1698)